src/matrixHelpers/FormioLoader/fixtures/index.js

```diff
@@ -0,0 +1,178 @@
+export const formioRes = {
```

Comment:
is it common pattern to just have it called 'fixtures' instead of 'test/fixtures' to denote that this is not production code? 

Coming from a java background, I do prefer how they don't 'mix' production code, test code, doc in the same tree structure.

Reply:
for me i used to the component-driven apprroach for UI development, where the folder of a UI component will contain  all the related code (base code, test, style, stories)
eg. https://github.com/mui/material-ui/tree/master/packages/mui-base/src/BadgeUnstyled
and tests that integrating across multiple components will sit outside the component tree.
in this way components is easy to modulised and decompose by service like https://bit.dev/ (though we don't need to worry about this)